### ISSUE TITLE
Apps page counter and apps page search not working

### ISSUE THREAD

**Issue by VladimirZD** [datagovuk/dgu-vagrant-puppet#76]
Hi, out apps page doesn't show number of apps
![image](https://cloud.githubusercontent.com/assets/8065995/6148946/86d492e2-b205-11e4-9329-f9b9415cd460.png)

And when i go to apps search (/search/everything/?f[0]=bundle%3Aapp) no results..
We have entered few apps..

I noticed that search isn't working at all when i try this /data/search?q=a  i get no results.

So i am thinking solar isn't setup right...

Any ides where to look?

Thnx

**Comment by tbalaz** [comment]
We have looked at Solr logging and found:

org.apache.solr.common.SolrException: Document is missing mandatory uniqueKey field: index_id

Any suggestions?

**Comment by VladimirZD** [comment]
Any1 has any idea?

Thnx

**Comment by davidread** [comment]
Yes sorry, the SOLR core for Drupal is not setup yet. I'll collect this thread together here: https://github.com/datagovuk/dgu-vagrant-puppet/issues/27

**Comment by VladimirZD** [comment]
Thnx. Do you have any ideas when you will get to it? Or any hints form us to fix it ? At moment it is pretty big issue :)

**Comment by VladimirZD** [comment]
Any change we get any hints for this one?

Thank you
